src/block/BaseSign.php

```diff
@@ -105,6 +150,32 @@ public function setText(SignText $text) : self{
 		return $this;
 	}
 
+	/**
+	 * Returns a color of Sign text.
+	 */
+	public function getTextColor() : Color{
+		return $this->textColor;
```

Comment:
I'd be happier if this was named "base text color" or something of that nature. The current name might be misleading to plugin devs (colour codes will still take effect; this only influences the default (unformatted) text colour).